### ISSUE TITLE
[508-defect-2] CST: Focus must be on error upon submission

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#41813]
## Description

While attempting to upload a file within the Claim Status Tool, the focus must be shifted to the input field with an error upon submission failure. The focus currently remains on the submit files for review button.

[Focus must be shifted to the input field with an error upon submission failure video](https://user-images.githubusercontent.com/14154792/169866049-113c0f47-9682-48ca-96d3-6db69bc535b6.MP4)
Related: https://github.com/department-of-veterans-affairs/va.gov-team/issues/41447 (under "Must" fix)

## Tasks

- [ ] Find and fix the focus issue shown in the video linked above
- [ ] Add tests

## Definition of done

- [ ] All tasks complete
- [ ] All tests passing

**Comment by saderagsdale** [comment]
@Mottie Should we remove the squad 2 label?

**Comment by joshkimux** [comment]
Opening this up again after testing to verify the fixes 🙇 

For cases where a user is uploading a document but doesn't select the document type, focus goes directly to the checkbox instead of the first error on the page which can result in screen reader users missing that there are more issues above. 

There's also an interesting bug where the error message for the checkbox will persist even after the user has selected it.

https://user-images.githubusercontent.com/14154792/178546743-ce07bf04-b315-4698-aaa4-b0931da0673d.mov

Before I ask for us to try and fix this, is this something out of our control? @jaredcunha and @Mottie

**Comment by Mottie** [comment]
We should be able to fix it

**Comment by jaredcunha** [comment]
I think I can see what's happening there, I take a look in the morning because signing in isn't working right now.

**Comment by jaredcunha** [comment]
@joshkimux - @jerekshoe and I looked into this a little after I saw the issue wasn't originally what I thought it was. It appears was not related to this issue/pr in particular, but it was related to migrating from `<Select>` to `<va-select>`, and that the error is now on a form field in the Shadow DOM.

There are two options in which we can proceed:
1. Revert back to the original `<Select>` component. This would be fastest option to do from a development perspective, but it's effectively kicking the can down the road since the old components would be deprecated and the issue may continue to exist in other implementations of `<va-select>`. 
2. Fix the web component. While it may fix the component, it's also significantly more work and might have unintended consequences with regard to wherever it is used. Also not sure how best to proceed there and who might be responsible. This should definitely be another ticket too.

What do you think? @jerekshoe anything else you want to add to this?

**Comment by joshkimux** [comment]
@jaredcunha @jerekshoe and I [connected on this on slack](https://dsva.slack.com/archives/C5AGLBNRK/p1657908544351819). We plan to:
- @jerekshoe to follow through with option 1
- @joshkimux to document that we'll want to switch back to va-select in the future and reach out to the VA Design System team to discuss fixing the web component
